task26.py

```diff
@@ -0,0 +1,15 @@
+# # Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+
+# # *Пример:*
+
+# # A = 3; B = 5 -> 243 (3⁵)
+# #     A = 2; B = 3 -> 8 
+
+def expon(a,b):
+    if b <= 1:
+        return a
+    else:
+        return a * expon(a, b - 1)
```

Comment:
```suggestion
    if b <= 1:
        return a

     return a * expon(a, b - 1)
```
![image](https://user-images.githubusercontent.com/49741194/224252116-d7fc4488-6a77-4fbd-bdcb-13f2ca3ac8d2.png)